aws_advanced_python_wrapper/resources/aws_advanced_python_wrapper_messages.properties

```
@@ -153,6 +153,40 @@ IamAuthPlugin.InvalidHost=[IamAuthPlugin] Invalid IAM host {}. The IAM host must
 IamAuthPlugin.IsNoneOrEmpty=[IamAuthPlugin] Property "{}" is None or empty.
 IamAuthUtils.GeneratedNewAuthToken=Generated new authentication token = {}
 
+LimitlessConnectionPlugin.FailedToConnectToHost=[LimitlessConnectionPlugin] Failed to connect to host {}.
+LimitlessConnectionPlugin.UnsupportedDialectOrDatabase=[LimitlessConnectionPlugin] Unsupported dialect '{}' encountered. Please ensure JDBC connection parameters are correct, and refer to the documentation to ensure that the connecting database is compatible with the Limitless Connection Plugin.
+
+LimitlessQueryHelper.UnsupportedDialectOrDatabase=[LimitlessQueryHelper] Unsupported dialect '{}' encountered. Please ensure JDBC connection parameters are correct, and refer to the documentation to ensure that the connecting database is compatible with the Limitless Connection Plugin.
+
+LimitlessRouterMonitor.ExceptionDuringMonitoringStop=[LimitlessRouterMonitor] Stopping monitoring after unhandled exception was thrown in Limitless Router Monitoring thread for node {}. Error: {}
```

```suggestion
LimitlessRouterMonitor.ExceptionDuringMonitoringStop=[LimitlessRouterMonitor] Stopping monitoring after unhandled error was thrown in Limitless Router Monitoring thread for node {}. Error: {}
```
Lets replace all instances of `exception` with `error` in the log messages